frontend/model/notifications/periodicNotifications.js

```diff
@@ -14,7 +14,7 @@ export const PERIODIC_NOTIFICATION_TYPE = {
 }
 
 const every1MinTimeout = { notifications: [], state: {}, delay: MINS_MILLIS }
-const every5MinTimeout = { notifications: [], state: {}, delay: 5 * MINS_MILLIS }
+const every5MinTimeout = { notifications: [], state: {}, delay: 0.15 * MINS_MILLIS }
```

Comment:
What happened here? This seems like a mistake maybe?

Reply:
yes, I think I forgot to change it back after testing the periodic notification. will fix it.